requirements.txt

```diff
@@ -9,8 +9,9 @@ uvloop>=0.14.0
 grpcio-tools>=1.33.2
 protobuf>=3.12.2
 configargparse>=1.2.3
-multiaddr>=0.0.9
-pymultihash>=0.8.2
+py-multihash>=0.2.3
+multiaddr @ git+https://github.com/multiformats/py-multiaddr.git
 cryptography>=3.4.6
 pydantic>=1.8.1,<2.0
 packaging>=20.9
+py-cid>=0.3.0
```

Comment:
If this package is not directly used, there is no need to specify it as a dependency